site/docs/plugins/autoquote.md

```diff
@@ -0,0 +1,58 @@
+# Always replying to messages
+
+Sometimes, especially for bots that are meant to be used in groups, it is necessary to always send messages as a reply (or quote) to the message that started the interaction.
+Usually, the way to do that is to mannualy add `reply_to_message_id` to the params of the mathod that sends the message (`sendText` / `reply`, `sendPhoto` / `replyWithPhoto`).
+However, if you're doing this for every message, it can get a bit messy and tiring.
+
+This plugin sets the value of `reply_to_message_id` param to `ctx.msg.message_id` for every `send` method (except for `sendChatAction`, which does not support this parameter), thus making every message a reply to the message that triggered that update.
+
+## Usage
+
+### For a single route
+
+Use this if you want all messages sent from within a specific context (like a specific command) to
+
+```ts
+import { Bot } from "grammy";
+import { addReplyParam } from "@roziscoding/grammy-autoquote";
+// import { addReplyParam } from 'https://deno.land/x/grammy_autoquote/mod.ts'
+
+const bot = new Bot("");
+
+bot.command("demo", async (ctx) => {
+  ctx.api.config.use(addReplyParam(ctx));
+  ctx.reply("Demo command!"); // This will quote the user's message
+});
+
+bot.start();
+```
+
+### Usage for every route
```

Comment:
Oops.. I missed this one. Title case :)

Reply:
```suggestion
### Usage for Every Route
```
Isn't it possible to do this vis css tho?